app/webhook.go

```diff
@@ -145,15 +150,27 @@ func (a *App) TriggerWebhook(payload *model.OutgoingWebhookPayload, hook *model.
 	}
 }
 
-func (a *App) doOutgoingWebhookRequest(url string, body io.Reader, contentType string) (*model.OutgoingWebhookResponse, error) {
-	req, err := http.NewRequest("POST", url, body)
+func (a *App) doOutgoingWebhookRequest(url string, body io.Reader, contentType, secret string) (*model.OutgoingWebhookResponse, error) {
+	bodyBytes, err := ioutil.ReadAll(body)
+	if err != nil {
+		return nil, errors.New("OutgoingWehookRequest payload failed to read.")
```

Comment:
Can we `errors.Wrap` here? Also, let's start with a lower-case error message to be idiomatic (i.e. `failed to read outgoing webhook request payload`)

Reply:
Good point, I used fmt.Errorf to wrap instead of importing a third party errror module.